apps/web/playwright/booking/longTextQuestion.e2e.ts

```diff
@@ -52,432 +52,4 @@ test.describe("Booking With Long Text Question and Each Other Question", () => {
     await bookingPage.cancelBooking(eventTypePage);
     await bookingPage.assertBookingCanceled(eventTypePage);
   });
-
-  test.describe("Booking With Long Text Question and Checkbox Group Question", () => {
```

Comment:
We now have unit tests wrapping logic around required fields so these bulky E2E tests are no longer needed